README.md

```diff
@@ -16,7 +16,7 @@ Repository to describe, develop, document and test the Identity And Consent Mana
 
 ## Release Information
 
-* `NEW`: Pre-release 0.2.0-rc.1 of guidelines and documentation for CAMARA APIs is available with the [v0.2.0-rc.1 tag](https://github.com/camaraproject/IdentityAndConsentManagement/tree/v0.2.0-rc.1).
+* `NEW`: Pre-release 0.2.0-rc.2 of guidelines and documentation for CAMARA APIs is available with the [r0.2.0-rc.2 tag](https://github.com/camaraproject/IdentityAndConsentManagement/tree/r0.2.0-rc.2).
```

Comment:
```suggestion
* `NEW`: Pre-release 0.2.0-rc.2 of guidelines and documentation for CAMARA APIs is available with the [r0.2.0-rc.2 tag](https://github.com/camaraproject/IdentityAndConsentManagement/tree/r0.2.0-rc.2). It contains the current version of the documents which are relevant for Fall24 Meta-Release, including:
  * [CAMARA APIs access and user consent management](https://github.com/camaraproject/IdentityAndConsentManagement/blob/r0.2.0-rc.2/documentation/CAMARA-API-access-and-user-consent.md)
  * [Camara Security and Interoperability Profile](https://github.com/camaraproject/IdentityAndConsentManagement/blob/r0.2.0-rc.2/documentation/CAMARA-Security-Interoperability.md)
```

Reply:
Fixed in https://github.com/camaraproject/IdentityAndConsentManagement/commit/ab5ec3c1bd4114c07a4f33e8bbd9c047ac1ebf7e